pkg/prompts/prompts.go

```diff
@@ -127,6 +143,65 @@ func validateBiggerThanZero(input string) error {
 	return nil
 }
 
+func (r *realPrompter) CaptureListDecision(
+	prompter Prompter, // we need this in order to be able to run mock tests
```

Comment:
same, need more descriptive names and comments

Reply:
Comments yes. Done. The comments make clear why the names are such.

Reply:
Comments yes. Done. The comments make clear why the names are such.